core/src/org/javarosa/core/model/instance/TreeReference.java

```diff
@@ -43,8 +43,9 @@
     // Multiplicity demarcates the position of a given element with respect to
     // other elements of the same name.
 
-    // Means there's a path to a node, but we haven't yet demarcated the
-    // multiplicity
+    // Since users usually want to select the first instance from the nodeset
+    // returned from a reference query, let the default multiplicity be
+    // selecting the first node.
```

Comment:
Sounds correct. Alternatively if it would make this clearer we could do something like

DEFAULT_MULTIPLICITY = MULTIPLICITY_FIRST;